cmd/arc_summary/arc_summary.py

```diff
@@ -705,8 +705,8 @@ def _l2arc_summary(Kstat):
         )
         sys.stdout.write("\n")
 
-        if arc["l2_arc_evicts"]['lock_retries'] + \
-                arc["l2_arc_evicts"]["reading"] > 0:
+        if arc["l2_arc_evicts"]['lock_retries'] != '0' or \
+           arc["l2_arc_evicts"]["reading"] != '0':
```

Comment:
> A seemingly obvious alternative would be to make the dictionary value be the integer value

This definitely seems like the right architectural fix.  Am I right in assuming you opted for this fix instead because it was the least disruptive to the existing code?